### PR TITLE
[Draft] Correct Draft Edit Tooltip in preferences

### PR DESCRIPTION
**Pull request by carlopav**
Thank you for creating a pull request to contribute to FreeCAD! To ease integration, please confirm the following:

- [ ] Branch rebased on latest master `git pull --rebase upstream master`
- [ ] Unit tests confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [ ] Commit message is [well-written](https://chris.beams.io/posts/git-commit/)
- [ ] Commit message includes `issue #<id>` or `fixes #<id>` where `<id>` is the [associated MantisBT](https://freecadweb.org/wiki/tracker#GitHub_and_MantisBT) issue id if one exists

And please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [0.19 Changelog Forum Thread](https://forum.freecadweb.org/viewtopic.php?f=10&t=34586).

---

### CODE REVIEW

File: src/Mod/Draft/Resources/ui/preferences-draftsnap.ui [hunk 1]
```diff
@@ -527,30 +518,21 @@
    <item>
     <widget class="QGroupBox" name="verticalGroupBox">
      <property name="toolTip">
-      <string>Draft edit preferences</string>
+      <string>Draft Edit gui command preferences</string>
```
Comment (luzpaz) [inline src/Mod/Draft/Resources/ui/preferences-draftsnap.ui:521]: Should gui here be explicit "Graphical User Interface" or at least capitalized 'GUI' ?

Comment (vocx-fc) [reply src/Mod/Draft/Resources/ui/preferences-draftsnap.ui:521]: In FreeCAD, the phrase "Gui Command" basically describes every button and menu entry which we can click on, because all such tools are based on the C++ class `Gui::Command`. So, in this case, Carlo is using the technical term. However, I think we shouldn't use such technical descriptions in user documentation because it may be confusing.

So, I would reword this to be simpler, like "Draft Edit preferences", or "Preferences for the Draft Edit tool".

Comment (carlopav) [reply src/Mod/Draft/Resources/ui/preferences-draftsnap.ui:521]: Ok, I probably agree. Let's go with "Draft Edit preferences"

**Comment by carlopav** [comment]
thanks @luzpaz and @vocx-fc for reviewing! @yorikvanhavre this is ready to be merged!
